package.json

```diff
@@ -33,10 +39,24 @@
     "glob-all": "^3.2.1",
     "html-loader": "^1.1.0",
     "html-webpack-plugin": "^4.3.0",
+    "ignore-loader": "^0.1.2",
+    "karma": "4.2.0",
+    "karma-chrome-launcher": "2.2.0",
```

Comment:
We don't use Chrome Headless, so you can remove it to avoid downloading unnecessary packages.